src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -41,8 +41,8 @@ private class ACCT_IndividualAccounts_TEST {
     */
     @isTest
     public static void insertContactTriggerOff() {
-        UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getAdminAccRecTypeID()));
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                        (Account_Processor__c = UTIL_Describe_API.getAdminAccRecTypeID()));
```

Comment:
I am interested in the reason behind moving 'Account_Processor__c' to a different line.

Reply:
@xinnanlisf , I think it was more of a consistency thing across all test methods. I can move it back.